Documentation.app/Contents/MacOS/Generation examples.playground/Pages/Parameters.xcplaygroundpage/Contents.swift

```diff
@@ -7,4 +7,132 @@
 /*:
  # Parameters
  
+ Network requests are usually parameterized. Clients must pass values in the network calls to get certain responses. These parameters can be included in different places in the network request: in the path, in the query parameters, in the request body or in the headers.
+ 
+ ## Path parameters
+ 
+ Path parameters are included in the path of the request. They are described in OpenAPI like:
+ 
+ ```yaml
+ openapi: "3.0.0"
+ info:
+   title: My Store
+   version: "1.0.0"
+   /{category}/product:
+     get:
+       tags:
+         - Product
+       operationId: getProductsForCategory
+       parameters:
+         - name: category
+           in: path
+           required: true
+           schema:
+             type: string
+       responses:
+         '200':
+           description: All products of a category
+           content:
+             application/json:
+               schema:
+                 $ref: '#/components/schemas/Products'
+ ```
+ 
+ Here, the parameter `category` is sent as part of the path of the request. When code is generated, it will be a parameter for the `getProductsForCategory` method, and the generated implementation will know where to encode it.
+ 
+ ## Query parameters
+ 
+ Query parameters are included after the path of the request, as a list of key-value pairs. They are described in OpenAPI like:
+ 
+ ```yaml
+ openapi: "3.0.0"
+ info:
+   title: My Store
+   version: "1.0.0"
+ paths:
+   /products:
+     get:
+       tags:
+         - Product
+       operationId: getProducts
+       parameters:
+         - name: sortBy
+           in: query
+           required: true
+           schema:
+             $ref: ' #/components/schemas/SortBy'
+       responses:
+         '200':
+           description: All products in the store
+           content:
+             application/json:
+               schema:
+                 $ref: '#/components/schemas/Products'
+ ```
+ 
+ The `sortBy` parameter is sent as a pair after the path of the request. When code is generated, it will be a parameter for the `getProducts` method, and the generated implementation will know where to encode it.
+ 
+ ## Body parameters
+ 
+ Body parameters are included in the body of a request. They are described in OpenAPI like:
+ 
+ ```yaml
+ openapi: "3.0.0"
+ info:
+   title: My Store
+   version: "1.0.0"
+ paths:
+   /customers/new:
+     post:
+       tags:
+         - Customer
+       operationId: newCustomer
+       requestBody:
+         required: true
+         content:
+           application/json:
+             schema:
+               $ref: '#/components/schemas/Customer'
+       responses:
+         '204':
+           description: Customer created
+ ```
+ 
```

Comment:
Maybe, we should add content will be encoded using JSON - and for now, it is the only supported encoding.